packages/lib/src/components/AmazonPay/defaultProps.ts

```diff
@@ -1,16 +1,19 @@
-export default {
+import {AmazonPayElementProps} from "./types";
+
+const defautProps: Partial<AmazonPayElementProps> = {
     cancelUrl: typeof window !== 'undefined' ? window.location.href : '',
     configuration: {},
     environment: 'TEST',
     locale: 'en_GB',
     placement: 'Cart',
     productType: 'PayAndShip',
-    region: 'EU',
     returnUrl: typeof window !== 'undefined' ? window.location.href : '',
```

Comment:
Does it make sense to give deprecation message that this property shouldn't be used (in case it is added in the AmazonPay config)? 

I can imagine merchants using it, then upgrading the library and out of the sudden it stops working. Is this a valid use case?

Reply:
There is no real reason for them to be using this prop. First there where no merchants using AmazonPay in the US since it lunched recently. Plus it has been broken since launch since it's not the prop that is sent to AmazonPay SDK.